src/LcdMenu.h

```diff
@@ -251,26 +251,6 @@ class LcdMenu {
         return currentMenuTable[cursorPosition + 1].getType() ==
                MENU_ITEM_END_OF_MENU;
     }
-    /**
-     * Reset the display
-     * @param isHistoryAvailable indicates if there is a previous position
-     */
-    void reset(boolean isHistoryAvailable) {
-        if (isHistoryAvailable) {
-            cursorPosition = previousCursorPosition;
-            top = previousTop;
-            bottom = previousBottom;
-        } else {
-            previousCursorPosition = cursorPosition;
-            previousTop = top;
-            previousBottom = bottom;
-
-            cursorPosition = 1;
-            top = 1;
-            bottom = maxRows;
-        }
-        update();
-    }
```

Comment:
Isn't it better to create another reset function with no parameters and make that one public instead?
Like...
```c++
void reset() {
    this->reset(false);
}
``` 
By default, reset not triggered by the library code should not consider history

Reply:
I use this in my code:

```c++
// reset menu to first item
menu.reset(false);
```

which wasn't possible before because it was private. Your solution sounds confusing to me though, I would suggest to use separate names, e.g. `resetMenu` for the public one?